src/LineUI/LineUnit.tsx

```diff
@@ -93,19 +93,21 @@ interface ILineUnitProps {
   moveEndCB?: () => void;
   label?: string;
   styling?: string;
+  bothDirectionMark: boolean;
+  hideMoveHandle?: boolean;
 }
 
 
 
 const LineUnit : React.FC<ILineUnitProps> = (props) => {
-  const { x1, y1, x2, y2, unit, lineMoveCallback, lineMoveStartCallback, options, lineSetId, label, styling='primary', moveEndCB=()=>{}} = props;
+  const { x1, y1, x2, y2, unit, lineMoveCallback, lineMoveStartCallback, options, lineSetId, label, styling = 'primary', moveEndCB = () => { }, bothDirectionMark, hideMoveHandle = true } = props;
   const { handleFinderActive, revealSetIndex, showMoveHandle, setIndexOffset, showDirectionMark} = options;
 
   const a = x1 - x2;
   const b = y1 - y2;
   const distance = Math.sqrt( a*a + b*b );
   //this distance 60 doesn't work now...
-  const hideGrabHandle = showMoveHandle === false || (showMoveHandle !== true && distance < 60);
+  const hideGrabHandle = hideMoveHandle ? (showMoveHandle === false || (showMoveHandle !== true && distance < 60)) : !hideMoveHandle;
```

Comment:
Change hideMoveHandle variable as per previous comment

Reply:
showMoveHandle is already available in the global option.

Reply:
You can give different name to it but hideMoveHandle is opposite of global prop showMoveHandle which is confusing here.  You can change hideMoveHandle to showLineMoveHandle or something similar.

Reply:
following up with the idea of the overrideShowMoveHandle  we can simplify this logic
// since the distance is not been accounted  now it can be removed if you prefer commenting the previous value as reference is fine
// showMoveHandle === false || (showMoveHandle !== true && distance < 60);

The logic I imagine that will be if either value is false it should be true to hide.

const hideGrabHandle = ( !showMoveHandle || !overrideShowMoveHandle)

Please consider and verify.